.mvn/maven.config

```diff
@@ -1,3 +1,3 @@
--Drevision=3.4.0
```

Comment:
not a bid deal but I think we are skipping 3.5.0 this way

Reply:
oh shoot